spec.html

```diff
@@ -5219,6 +5219,17 @@
             </tr>
             <tr>
               <td>
+                [[GlobalThisValue]]
+              </td>
+              <td>
+                Object
+              </td>
+              <td>
+                The value returned by `this` in global scope. This may be the same as the global object, or may not. For example, in HTML the `this` value is a `WindowProxy` object, whereas the global object is a `Window`.
```

Comment:
Also, I'm not sure "usually the same as" is a good sentence fragment given that in reality 95% of InitializeHostDefinedRealm() calls will use divergent global and `this` value. (I think an estimate of 5% for Node.js is generous personally...)